plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarImpl.kt

```diff
@@ -138,20 +125,37 @@ class ScaleBarImpl : ScaleBar, View {
       mapViewWidth = mapViewWidth
     }
 
+  private var currentSegmentsConfiguration: SegmentsConfiguration? = null
+
   /**
    * distancePerPixel in current location latitude
    */
   override var distancePerPixel = 0F
     set(value) {
       if (field != value) {
+        // TODO: Check the value is always different than field for Imperial units due to this conversion
```

Comment:
do we want to actually merge it with following todo? perhaps we can create the Jira ticket and mention it here instead

Reply:
I just fixed it.